lms/djangoapps/discussion_api/tests/test_views.py

```diff
@@ -634,6 +634,16 @@ def test_error(self):
         response_data = json.loads(response.content)
         self.assertEqual(response_data, expected_response_data)
 
+    def test_unsupported_media_type(self):
```

Comment:
Can you create a new Test Mixin and put `test_unsupported_media_type` in it so it doesn't have to be duplicated across all Views that have/will have Patch methods?